File: src/Entity/Cours.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;
use phpDocumentor\Reflection\Types\Float_;
use phpDocumentor\Reflection\Types\Integer;
use Symfony\Component\Validator\Constraints as Assert;
use Symfony\Component\Serializer\Annotation\Groups;


/**
 * Cours
 *
 * @ORM\Table(name="cours")
 * @ORM\Entity
 * @ORM\Entity(repositoryClass="App\Repository\CoursRepository")
 */
class Cours
{
    /**
     * @var int
     *
     * @ORM\Column(name="idC", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     * @Groups("c:read")
     */
    private $idc;

    /**
     * @var string
     * @ORM\Column(name="nom", type="string", length=50, nullable=false)
     * @Assert\Length(min=3)
     * @Assert\NotBlank
     * @Groups("c:read")
     */
    private $nom;

    /**
     * @var string
     *
     * @ORM\Column(name="type", type="string", length=200, nullable=false)
     * @Groups("c:read")
     */
    private $type;

    /**
     * @var string
     *
     * @ORM\Column(name="description", type="string", length=500, nullable=false)
     * @Assert\NotBlank
     * @Groups("c:read")
     */
    private $description;

    /**
     * @var string
     *
     * @ORM\Column(type="string")
    @Assert\File(
     *     maxSize = "1024k",
     *     mimeTypes = {"application/pdf", "application/x-pdf"},
     *     mimeTypesMessage = "Please upload a valid PDF"
     * )
     */
    private $cours;

    /**
     * @var string
     * @ORM\Column(name="prix", type="string", length=50, nullable=false)
     * @Groups("c:read")
     */
    private $prix;

    /**
     * @var int
     * @ORM\Column(name="rate", type="integer", nullable=true)
     */
    private $rate;


    public function getIdc(): ?int
    {
        return $this->idc;
    }

    public function getNom(): ?string
    {
        return $this->nom;
    }

    public function setNom(string $nom): self
    {
        $this->nom = $nom;

        return $this;
    }

    public function getType(): ?string
    {
        return $this->type;
    }

    public function setType(string $type): self
    {
        $this->type = $type;

        return $this;
    }

    public function getDescription(): ?string
    {
        return $this->description;
    }

    public function setDescription(string $description): self
    {
        $this->description = $description;

        return $this;
    }

    public function getCours()
    {
        return $this->cours;
    }

    public function setCours(string $cours)
    {
        $this->cours = $cours;

        return $this;
    }

    public function getPrix(): ?string
    {
        return $this->prix;
    }

    public function setPrix(string $prix): self
    {
        $this->prix = $prix;

        return $this;
    }

    /**
     * @return int
     */
    public function getRate()
    {
        return $this->rate;
    }

    /**
     * @param int $rate
     */
    public function setRate(int $rate): void
    {
        $this->rate = $rate;
    }





}

File: Mobileaziz/CodePI/src/com/mycompany/myapp/entities/Cours.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycompany.myapp.entities;

/**
 *
 * @author asus
 */
public class Cours {
    private int idc;
    private String nom;
    private String type;
    private String description;
    private String cours;
    private String prix;
    private int rate;

    public Cours(int idc, String nom, String type, String description, String cours, String prix, int rate) {
        this.idc = idc;
        this.nom = nom;
        this.type = type;
        this.description = description;
        this.cours = cours;
        this.prix = prix;
        this.rate = rate;
    }
    
    

    public Cours(String nom, String description, String prix, int rate) {
        this.nom = nom;
        this.description = description;
        this.prix = prix;
        this.rate = rate;
    }

    public Cours() {
    }

    public int getIdc() {
        return idc;
    }


 
   public void setIdc(int idc) {
        this.idc = idc;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getCours() {
        return cours;
    }

    public void setCours(String cours) {
        this.cours = cours;
    }

    public String getPrix() {
        return prix;
    }

    public void setPrix(String prix) {
        this.prix = prix;
    }

    public int getRate() {
        return rate;
    }

    public void setRate(int rate) {
        this.rate = rate;
    }

    @Override
    public String toString() {
        return "Cours{" + "nom=" + nom + ", type=" + type + ", description=" + description + ", prix=" + prix + ", rate=" + rate + '}';
    }

    public Cours(String nom, String type, String description, String cours, String prix, int rate) {
        this.nom = nom;
        this.type = type;
        this.description = description;
        this.cours = cours;
        this.prix = prix;
        this.rate = rate;
    }

   
    
    
}
